### PR TITLE
Update associations with many to many

### PR DESCRIPTION
**Pull request by catherinemdean15**
Co-authored-by: Isabelle Villasenor <45639675+isabellevillasenor@users.noreply.github.com>

### Description
-Update associations to clear hasMany error

### CODE REVIEW

File: code/api/src/config/database.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "username": "isabellevillasenor",
+    "username": "catherinedean",
```
Comment (gaj23) [inline code/api/src/config/database.json:3]: We will each need to change it on our local .gitignore

**Review by gaj23 — APPROVED** [review]
Thank you for your speedy response & changes!

File: code/api/src/modules/crate/model.js [hunk 2]
```diff
@@ -13,9 +13,10 @@ module.exports = function(sequelize, DataTypes) {
 // Define relationship here as you would in Models in Rails
   Crate.associate = function(models) {
     Crate.hasMany(models.Subscription)
-    Crate.hasMany(models.crateProducts)
-    Crate.hasMany(models.Products).through(models.crateProducts)
-  }
```
Comment (srslie) [inline code/api/src/modules/crate/model.js:18]: Wizards, thanks for fixing that hasMany error.

File: code/api/src/modules/product/model.js [hunk 3]
```diff
@@ -24,12 +24,14 @@ module.exports = function(sequelize, DataTypes) {
     }
   })
 
-  Products.associate = function(models) {
-    Products.hasMany(models.crateProducts)
-    Products.hasMany(models.Crate).through(models.crateProducts)
-    Products.hasMany(models.userProducts)
-    Products.hasMany(models.User).through(models.userProducts)
-  }
+  Product.associate = function(models) {
```
Comment (srslie) [inline code/api/src/modules/product/model.js:27]: Alright sequelize!

**Review by srslie — APPROVED** [review]

**Comment by rachelbuchta** [comment]
Awesome work trouble shooting this!! What a journey with many manys

**Review by tashiad — APPROVED** [review]
Looks great, thanks for working so hard to fix this!

**Review by rachelbuchta — APPROVED** [review]

**Pull request merged**